### ISSUE TITLE
PaxDetail - wrong fab actions shown after reopening hit status

### ISSUE THREAD

**Issue by gtasautumn** [US-CBP/GTAS-UI#537]
**Describe the bug**
Once a passenger with a hit has their status set to `RE_OPENED`, the fab action **Set to RE-OPENED** remains visible. This should be replaced by the **Create Manual Hit** option.

**To Reproduce**
1. Go to FlightPax
2. Click a user with manual hits
3. on PaxDetails, open the green fab button
4. Click on **Set to RE-OPENED**
5. Refresh the page
6. open the green fab button again
7. Verify that the **Set to RE-OPENED** option is still visible
8. In the DB, **hit_view_status** table, verify that the hv_status = "RE-OPENED" where hv_passenger_id = your passenger

**Expected behavior**
The option to set the hit status to re-opened should not appear if the status = re-opened. We should see the option to add a new manual hit or set the status to reviewed. We should probably discuss this.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by dje04001** [comment]
I agree with you. It's worth having a followup on this to make sure everything works as intended.
